programs/flobrij/src/lib.rs

```diff
@@ -53,6 +53,10 @@ pub mod flobrij {
         Ok(())
     }
 
+    pub fn delete_receipt(_ctx: Context<DeleteReceipt>) -> ProgramResult {
```

Comment:
Hey @art1313 would you mind adding a unit test for deleting receipt?

Here's the one I wrote for deleting support levels: https://github.com/flobrij/rust-programs/blob/main/tests/supportlevel.test.ts#L102

I have a separate test file for Receipts here: `tests/receipt.test.ts`.

Thank you, and let me know if you have any questions.

Reply:
Thanks, I have added unit test for receipt.

Reply:
Thanks @art1313 for adding tests!